felix/usagerep/usagerep_test.go

```diff
@@ -64,7 +63,7 @@ var _ = Describe("UsageReporter with mocked URL and short interval", func() {
 		configUpdateC = make(chan map[string]string)
 
 		// Create a usage reporter and override its base URL and initial interval.
-		u = New(StaticItems{KubernetesVersion: "v1.17.0"}, 500*time.Millisecond, 1*time.Second, statsUpdateC, configUpdateC)
+		u = New(StaticItems{KubernetesVersion: "v1.32.2"}, 500*time.Millisecond, 1*time.Second, statsUpdateC, configUpdateC)
```

Comment:
Did you mean v1.23? I don't think the version actually matters here in the test, though.

Reply:
Oh yes, that's some typo. Will correct that.